edc-extensions/control-plane-adapter-api/src/main/java/org/eclipse/tractusx/edc/api/cp/adapter/dto/NegotiateEdrRequestDto.java

```diff
@@ -23,16 +23,17 @@
 import java.util.List;
 
 import static org.eclipse.edc.spi.CoreConstants.EDC_NAMESPACE;
+import static org.eclipse.tractusx.edc.edr.spi.CoreConstants.TX_NAMESPACE;
 
 public class NegotiateEdrRequestDto {
```

Comment:
I know that in EDC we have this double-transformation `JsonObject` -> `Dto` -> entity, but that was done as a temporary solution, so I think here that detour over the Dto is not needed and we can just convert `JsonObject` -> entity

Reply:
I did use the DTO ways since i  need also transformer 

- ContractOffer 
- CallbackAddress

And there is not tranformer JsonObjectTo for the above entities available in the upstream. Since the request is similar to `NegotiationInitiateRequestDto` i basically did the same here

Reply:
the double-transformation in the upstream will eventually go away, so we should not replicate that behaviour here. For example the `EndpointDataReferenceEntryDto` is only ever used as an intermediary, and it only contains 3 strings...

Reply:
The `EndpointDataReferenceEntryDto` has been removed. `NegotiationInitiateRequestDto` will be removed in separated PR

Reply:
https://github.com/eclipse-tractusx/tractusx-edc/issues/398